### ISSUE TITLE
Logstash Kafka Beta "add_field" not working

### ISSUE THREAD

**Issue by sanjay-varanasi** [logstash-plugins/logstash-input-kafka#101]
We're using Logstash's (version 2.3.2) 3.0.0.beta3 Kafka plugin to read from Kafka topic(s). I'm trying to add an "add_field" to the Kafka input so I can redirect the output based on this field. However, when I look at the stdout with metadata set to true, I don't see the custom field I created. Only the default "@version" and "@timestamp" fields. I'm guessing this is a bug, but I have a history of making silly mistakes. My config file:

```
input
{
 kafka
 {
  bootstrap_servers => "<snipped>"

  topics => "newkafka"
  ssl => true
  ssl_keystore_location => "/opt/logstash/ssl/server.keystore.jks"
  ssl_keystore_password => "<snipped>"
  ssl_truststore_location => "/opt/logstash/ssl/server.truststore.jks"
  ssl_truststore_password => "<snipped>"
  add_field => { "[@metadata][kafka]" => "logstash" }
 }
}


output
{

 stdout 
 {
  codec => rubydebug { metadata => true }
 }
}


```

And this is the output I'm seeing

```
{
       "message" => "Metadata Test",
      "@version" => "1",
    "@timestamp" => "2016-06-22T17:20:59.844Z"
}

```

Help!

**Comment by sanjay-varanasi** [comment]
It isn't just metadata that isn't working, though. Trying to create a non-metadata field also doesn't seem to work. Tried this, and same result (or lack of it)

`add_field => { "topic" => "newkafka" }`

Output:

```
{
       "message" => "Metadata Test",
      "@version" => "1",
    "@timestamp" => "2016-06-22T17:20:59.844Z"
}
```

**Comment by talevy** [comment]
this was fixed in `3.0.2` release. go ahead and update to that plugin and this should be fixed. let me know how it goes!

**Comment by sanjay-varanasi** [comment]
Excellent! 3.0.2 definitely fixes the issue for me. 

By the way, where may I track release information for this plugin? I had no idea that the plugin was out of Beta! Thanks!
